src/otx/algo/utils/support_otx_v1.py

```diff
@@ -97,8 +97,7 @@ def load_seg_lite_hrnet_ckpt(state_dict: dict, add_prefix: str = "") -> dict:
         """Load the OTX1.x lite hrnet segmentation checkpoints."""
```

Comment:
@sungmanc, do we still need this function `load_seg_lite_hrnet_ckpt`?

Reply:
Now, we could use the `load_common_ckpt` for this function. However, it's still needed to remain due to the different patterns between `SegNext` and `LitHRNet`.

```python3 
@staticmethod
    def load_seg_lite_hrnet_ckpt(state_dict: dict, add_prefix: str = "") -> dict:
        """Load the OTX1.x lite hrnet segmentation checkpoints."""
        for key in list(state_dict.keys()):
            val = state_dict.pop(key)
            state_dict[add_prefix + key] = val
        return state_dict
###########################################################
    def load_seg_lite_hrnet_ckpt(state_dict: dict, add_prefix: str = "") -> dict:
        """Load the OTX1.x lite hrnet segmentation checkpoints."""
        return OTXv1Helper.load_common_ckpt(state_dict, add_prefix)
```

Reply:
Why not just directly calling `OTXv1Helper.load_common_ckpt(state_dict, add_prefix)` from `LiteHRNet`?

Reply:
We also can do what you suggested. I'll patch this part when I check the mobilenetv3 backward compat.